app.rb

```diff
@@ -6,25 +6,23 @@
 require './rental'
 require './classroom'
 require './create_person'
+require 'json'
+require 'byebug'
 
 class App
   include CreatePerson
   attr_reader :books, :persons, :rentals
 
   def initialize
-    @books = []
-    @persons = []
-    @rentals = []
-  end
-
-  def list_books
-    if @books.empty?
-      puts 'No books found'
-    else
-      @books.each_with_index do |book, index|
-        puts "#{index}) Title: #{book.title}, Author: #{book.author}"
+    @books = load_json('books.json').map { |b| Book.new(b['title'], b['author']) }
+    @persons = load_json('people.json').map do |person|
```

Comment:
- [ ] The app will show an error in case the books.json, people.json, or rentals.json files do not exist. It would be be better to handle the case when one or more of the files are missing (the app should work and show no errors). To solve this problem I would recommend adding the following code that checks if a JSON file exists, if not it will create one: 
```rb
File.open('books.json', 'w') { |f| f.write JSON.generate([]) } unless File.exist? 'books.json'
```